cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/FileStreamAdmin.java

```diff
@@ -111,6 +111,7 @@ public void dropAllInNamespace(Id.Namespace namespace) throws Exception {
 
     for (final Location streamLocation : locations) {
       doTruncate(streamLocation);
+      alterExploreStream(StreamUtils.getStreamIdFromLocation(streamLocation), false);
```

Comment:
Should this also be added to drop()? and then in here just call drop() instead of truncate and alterExploreStream?